src/js/providers/html5.js

```diff
@@ -1164,10 +1160,10 @@ define([
                    track.mode = 'disabled';
                 });
             }
-            if(index > 0 && index <= _textTracks.length) {
+            if (index > 0 && index <= _textTracks.length) {
                 _currentTextTrackIndex = index - 1;
+                _videotag.setAttribute('crossorigin', 'anonymous');
```

Comment:
Tried to set it only when the tracks have cors issue, but track.src is undefined - textTracks does not have src in it.